new-client/src/plugins/Print/PrintModel.js

```diff
@@ -398,6 +398,9 @@ export default class PrintModel {
     const size = this.map.getSize();
```

Comment:
This should be in a pr of it's own right? Or does your change affect the print?

Reply:
Oh... no... Yes, you are right. Good find. Commited by misstake.